osu.Desktop/osu.nuspec

```diff
@@ -12,12 +12,12 @@
         <description>click the circles. to the beat.</description>
         <summary>click the circles.</summary>
         <releaseNotes>testing</releaseNotes>
-        <copyright>Copyright ppy Pty Ltd 2007-2017</copyright>
+        <copyright>Copyright ppy Pty Ltd 2007-2018</copyright>
         <language>en-AU</language>
     </metadata>
     <files>
       <file src="*.exe" target="lib\net45\" exclude="**vshost**"/>
-      <file src="*.dll" target="lib\net45\"/>
+      <file src="*.dll" target="lib\net45\"/>

```

Comment:
What's this?

Reply:
Didn't notice that, actually. diff doesn't recognize a difference when disabling trailing whitespace checks, which may have been caused by my editor auto-trimming it.

Reply:
```      <file src="*.exe" target="lib\net45\" exclude="**vshost**"/>^M
      <file src="*.dll" target="lib\net45\"/>
      <file src="*.config" target="lib\net45\"/>^M
```

Seems like old osu.nuspec was missing a CR to begin with, your editor must've automatically added it. Nothing wrong with this I guess

Odder is why AppVeyor doesn't work :P

Reply:
Could AppVeyor not work because my fork was behind some commits? I tried getting it up-to-date, but AppVeyor is still failing for some reason... Maybe my submodules aren't current